cranelift/codegen/src/isa/aarch64/abi.rs

```diff
@@ -85,7 +85,7 @@ impl ABIMachineSpec for AArch64MachineDeps {
         params: &[ir::AbiParam],
         args_or_rets: ArgsOrRets,
         add_ret_area_ptr: bool,
-    ) -> CodegenResult<(Vec<ABIArg>, i64, Option<usize>)> {
+    ) -> CodegenResult<(SmallVec<[ABIArg; 6]>, i64, Option<usize>)> {
```

Comment:
Can we put a `type ABIArgVec ` SmallVec<[ABIArg; 6]>` in `machinst::abi_impl` then use it here (and `ABIArgVec::new()` below)? That would allow us to tweak the size later more easily without updating each backend.